TODO.md

```diff
@@ -0,0 +1,23 @@
+## TODO
+
+### High Priority
+
+#### Support the new .conda package format.
```

Comment:
This should be good now?! :)

Reply:
Just removed! Sorry forgot to check the TODOs before I opened!

It would be good to get this merged soon - since I've rebased the slow_upload branch locally I'm not sure I can do it again without messing up the git history here - so if/when we merge I can create a fresh clone/branch to work off of.